packages/ruby/examples/metrics-rails/app/controllers/metrics_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'readme/webhook'
+
+class MetricsController < ApplicationController
+  def index
+    render json: { 'message' => 'hello world' }
+  end
+
+  def post
+    head :ok
+  end
+
+  def webhook
```

Comment:
it'd be nice to try to follow standard [naming conventions](https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e#controllers) if possible